.eslintrc.json

```diff
@@ -7,28 +7,24 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "next",
-    "next/core-web-vitals"
+    "next/core-web-vitals",
+    "prettier"
   ],
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
     "ecmaVersion": "latest",
     "sourceType": "module"
   },
   "rules": {
-    "indent": [
-      "error",
-      2
-    ],
+    "indent": "off",
```

Comment:
why turn off indent rule @aatbip ?

Reply:
`"indent" : "warn"` has been set in the next commit @saugatmaharjan